docs/tutorials/integrating-with-laravel.md

```diff
@@ -0,0 +1,85 @@
+# Integrating with Laravel
+
+This page explains how to integrate Shoelace with a [Laravel](https://laravel.com) app using a local Webpack bundle. This is a community-maintained document. For questions about this integration, please [ask the community](/resources/community).
+
+## Requirements
+
+This integration has been tested with the following:
+
+- Laravel >= 8
+- Node >= 14
+- Laravel Mix >= 6
+
+## Instructions
+
+These instructions assume an out-of-the-box [Laravel 8+ install](https://laravel.com/docs/8.x/installation), using [Laravel Mix](https://laravel.com/docs/8.x/mix) to compile assets.
+Be sure to run `npm install` to install the default Laravel front-end dependencies before installing Shoelace. 
+
+### Install the Shoelace package
+```bash
+npm install @shoelace-style/shoelace
+```
+
+### Import the Default Theme
+
+Import Shoelace's default theme (stylesheet) in `/resources/css/app.css`:
+
+```css
+@import "/node_modules/@shoelace-style/shoelace/dist/themes/light.css";
+```
+
+### Import Required Scripts
+
+Import each Shoelace component you plan to use in `/resources/js/boostrap.js`:
+
+```js
+import { SlAlert, SlAnimation, SlButton, ... } from '@shoelace-style/shoelace'
+```
+
+### Configure Laravel Mix
+[Laravel Mix](https://laravel.com/docs/8.x/mix) is a wrapper around Webpack that simplifies configuration. Mix is used by default for compiling front-end assets in Laravel.
+
+Modify `webpack.mix.js` to add Shoelace's assets to Webpack's build process
```

Comment:
We probably want to make sure `<sl-icon>` works using the default icon library (components that use icons internally use the `system` library, which doesn't require assets). You can test this by adding `<sl-icon name="house"></sl-icon>`. It probably won't render anything, so we'll need to add a call to `setBasePath` as described here:

https://shoelace.style/getting-started/installation?id=setting-the-base-path

Reply:
Sure. The house icon displayed without explicitly setting the basepath. I was only able to get the icons working again using `setBasePath("/")` -- does that sound right?

Reply:
Looking at line 46, it should probably be set to `/assets`. The base path basically needs to point to the same folder the assets get copied to on the server (but translated to URL).

Reply:
I initially tried `setBasePath("/assets");`, but that doesn't resolve properly: 
`/assets/assets/icons/house.svg:1 Failed to load resource: the server responded with a status of 404 (Not Found)`

Maybe it's the way I configured the copy in `webpack.mix.js`: 
`.copy("node_modules/@shoelace-style/shoelace/dist/assets", "public/assets")`
The entire set of Shoelace icons ends up in `/public/assets/icons`.